<launch>
    <param name="ORBvoc_address" type="string" value="$(find orbslam3)/Vocabulary/ORBvoc.txt" />
    <param name="yaml_address" type="string" value="$(find orbslam3)/Examples/Stereo/arpg_realsense_stereo.yaml" />
    <param name="do_rectification" type="bool" value="true" />
    <param name="show_orbslam_UI" type="bool" value="false" />
    
    <param name="im_left_topic" type="string" value="/gray_image0" />
    <param name="im_right_topic" type="string" value="/gray_image1" />
    <param name="im_left_mask_topic" type="string" value="/image_seg0" />
    <node type="ros_stereo" pkg="orbslam3" name="ros_stereo" output="screen">
    </node>
</launch>
